app/templates/gulpfile.js

```diff
@@ -949,10 +949,10 @@ function getNodeOptions(env) {
       '`gulp init-local` creates local.json which can be modified for other environments as well');
   }
   var port = args['app-port'] || process.env.PORT || envJson['node-port'] || config.defaultPort;
-  return {
-    script: config.nodeServer,
-    delayTime: 1,
-    env: {
+
+  console.log("config.nodeServer : " + config.nodeServer);
```

Comment:
Could you either make this a more elegant message (if this info is not yet printed somewhere yet), or just get rid of it?